.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#package-ecosystem-
```

Comment:
I personally prefer Renovatebot, although I haven't configured it for this repo yet. Could you please remove this file from this PR? Thanks!